test/format/ops.c

```diff
@@ -914,6 +927,12 @@ deadlock:			++tinfo->deadlock;
 			break;
 		}
 
+		/*
+		 * Clear the thread's read timestamp: it no longer needs to
+		 * be pinned.
+		 */
+		tinfo->commit_timestamp = 0;
```

Comment:
The comment says "read timestamp", but the code says `commit_timestamp` Should this be clearing both?

Reply:
> The comment says "read timestamp", but the code says `commit_timestamp` Should this be clearing both?

Thanks, Alex -- code was wrong, we should be clearing the read-timestamp here, at the end of the transaction.

I've pushed a fix.